qds_sdk/clusterv2.py

```diff
@@ -509,6 +517,12 @@ def cluster_info_parser(argparser, action):
                                    default=None,
                                    help="overrides for airflow cluster", )
 
+        internal_group = argparser.add_argument_group("internal settings")
```

Comment:
Cluster V2 API supports adding the image overrides if it is provided, and it doesnot require super admin access.